### ISSUE TITLE
[Bug] IPv6 RPC connections broken

### ISSUE THREAD

**Issue by altendky** [Chia-Network/chia-blockchain#11628]
### What happened?

https://github.com/Chia-Network/chia-blockchain/pull/11578 broke IPv6 RPC connections by hard coding `0.0.0.0` for the host at https://github.com/Chia-Network/chia-blockchain/pull/11578/files#diff-926449f427065cf100c0af5785ac61529c768ab4cccbef25ccb86e82074b753fR270.  Initial exploration started in https://github.com/Chia-Network/chia-blockchain/pull/11623.  Actual fix being worked in https://github.com/Chia-Network/chia-blockchain/pull/11764.

### Version

main: https://github.com/Chia-Network/chia-blockchain/commit/817baa3096ee7cb4f820bf04599a2d231faeff94

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

```shell
2022-05-24T14:53:31.336 full_node chia.rpc.util           : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 16, in inner
  File "chia\rpc\rpc_server.py", line 172, in open_connection
ValueError: Start client failed, or server is not set
```
```

**Comment by PsyDafke** [comment]
:+1:

**Comment by emlowe** [comment]
Fixed in #11764
